contrib/!lang/erlang-elixir/README.md

```diff
@@ -98,18 +122,19 @@ Note that `EDTS` is disable on Windows.
 <kbd>SPC m e r</kbd>  | Evaluate region
 <kbd>SPC m e R</kbd>  | Evaluate region and insert result
 
-#### Elixir REPL interactions
+##### REPL interactions
 
     Key Binding       |                 Description
 ----------------------|------------------------------------------------------------
 <kbd>SPC m s i</kbd>  | Start an `iex` inferior process
+<kbd>SPC m s I</kbd>  | Start an IEx process with mix (`iex -S mix`)
```

Comment:
If starting via Mix is the most used way to start an iteractive REPL then we could swap the bindings `SPC m s i` and `SPC m s I`.